README.md

```diff
@@ -1,60 +1,13 @@
-
-### Homework and Assignments
-* Homework will be hard and will take time to complete. Do not put it off. 
-* Start homework early and come to class with questions. Starting homework late is a recipe for disaster and pain.
-* All assignments, due dates, and grading information are in [**Assignments**](Assignments.md)
-* Homework will be done in pairs. If you are experiencing issues, please contact an instructor.
-* Code reviews are done individually, each student should review code that is not done by them or their partner.
-* The day homework is initially due we will typically go over the assignment in class. (no guarantees)
-
-## Computer Setup ##
-- [Visual Studio Code](https://code.visualstudio.com/) This can largely be used as an alternative to Visual Studio. Many of us also use it as our catch-all lightweight text/code editor.
-- [Azure for Students](https://azure.microsoft.com/en-us/free/students/) You will need to sign up for the free Azure student credits. This provides a place for us to deploy our application.
-The course work will be cross-platform. However, in class the instructors will be working on Windows in both VSCode (for front end) and Visual Studio (for back end).
-- [nodejs & npm (LTS)](https://nodejs.org/en/download/) Node Package Manager will provide tooling for bulding our front end web site.
-
-### Recommended ###
-- [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/). Version 17.5 or later. The community edition is fine, though we believe most students should have access to higher SKUs with EWU's MSDN (this assumption may be wrong). Lab computers should have Professional already installed. This will install .NET 7.0.
-- [GitKraken](https://gitkraken.keboo.dev/) Though you can do all of the git interaction from within Visual Studio or on the command line, GitKraken is free for open source work. It also provides a nice graphical version of the commit history so you can see how various commits and branches relate. 
-- [GitHub Student Developer Pack](https://education.github.com/students) Lots of great development tools and resources. The JetBrains products ReSharper (a plugin for Visual Studio), and Rider (a full C# IDE) are great tools for helping you write better code. 
-
-### VS Code Extensions used in class ###
-- [JEST](https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest) Unit testing extension
-- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) code formatting extension
-- [Vetur](https://marketplace.visualstudio.com/items?itemName=octref.vetur) Vue tooling
-- [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
+# Please press our buttons <br>
+# https://yellow-coast-01713141e.3.azurestaticapps.net/
+![](https://media.giphy.com/media/vFKqnCdLPNOKc/giphy.gif)
+<br>
+Extra credit pls. It was mentioned in class Meg likes cats.
```

Comment:
I do like cats. I have two. :) Giving you guys an extra +5 ontop of the +3 for the actual game on the game page.